src/Okta.DNX.OAuth.ResourceServer/Models/TodoRepository.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Okta.DNX.OAuth.ResourceServer.Models
+{
+    public class TodoRepository : ITodoRepository
+    {
+        private static ConcurrentDictionary<string, TodoItem> _todos =
+              new ConcurrentDictionary<string, TodoItem>();
+
+        public TodoRepository()
+        {
+            Add(new TodoItem { Name = "Item1" });
```

Comment:
Would be nice to add a comment here just saying "Start off the app with an example todo item" or something like that, if that's what we're doing here.